lib/save.js

```diff
@@ -0,0 +1,18 @@
+var thenify = require('thenify')
+var fs = require('fs')
+var writeFile = thenify(fs.writeFile)
+
+module.exports = function save (pkg, installedPackages, saveType) {
+  var packageJson = pkg.pkg
+  packageJson[saveType] = packageJson[saveType] || {}
+  installedPackages.forEach(function (dependency) {
+    // Assumes package name of in the format of "package-name@x.x.x"
+    // TODO: Support other install sources (Github, scoped pkgs etc.)
```

Comment:
hmm, the npm behavior is to do:

```
npm install --save rimraf
```

and it'll pick up `rimraf@^2.5.1` for you based on what it resolves. Might simply need to make `installMultiple()` throw data of the top-level packages it installed.